### PR TITLE
Relax numpy pinning to global pin for aarch64

### PR DESCRIPTION
**Pull request by step21**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by step21** [comment]
@conda-forge-admin please rerender

**Comment by step21** [comment]
@conda-forge-admin please rerender

**Comment by step21** [comment]
This is to manually fix https://github.com/conda-forge/statsmodels-feedstock/pull/29

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,8 +20,9 @@ requirements:
     - python
     - pip
     - cython
-    - numpy =1.15  # [py<38]
-    - numpy =1.16  # [py>=38]
+    - numpy  # [ aarch64 ]
+    - numpy =1.15  # [py<38 and not aarch64 ]
+    - numpy =1.16  # [py>=38 and not aarch64 ]
```
Comment (xhochy) [inline recipe/meta.yaml]: Small style nitpick
```suggestion
    - numpy  # [aarch64]
    - numpy =1.15  # [py<38 and not aarch64]
    - numpy =1.16  # [py>=38 and not aarch64]
```

**Comment by step21** [comment]
@conda-forge/statsmodels kinda anxious to merge this, please have a look / comment if possible.

**Comment by croth1** [comment]
@xhochy would you have capacities to do the reviewing here? I feel I don't have a good overview  what implications that change would have.

**Comment by xhochy** [comment]
Implications are minimal, the package can be installed on aarch64 then.

**Comment by step21** [comment]
@conda-forge-admin please restart ci

**Comment by step21** [comment]
@croth1 can you restart ci to fix the ppc error? Seems like just a usual travis/ppc glitch

**Pull request merged**